### PR TITLE
Added N to strings output for unicode handling on some collations

### PR DESCRIPTION
**Pull request by nathfy**
Hi, this sproc has been really useful for us in building seeddata for our tests. One thing I needed todo to get it working for us was to add `N` infront of the strings to fix the unicode inserts of text to NVARCHAR cols. This PR adds that in to the sproc.

Thanks for sharing it on github.

### CODE REVIEW

File: master.dbo.sp_generate_merge.sql [hunk 1]
```diff
@@ -666,4 +666,4 @@ GRANT EXEC ON sp_generate_merge TO public
 SET NOCOUNT OFF
 GO
 
-PRINT 'Done'
+END
```
Comment (dnlnln) [inline master.dbo.sp_generate_merge.sql]: This is causing an error when trying to deploy the proc:
```
Msg 102, Level 15, State 1, Line 668
Incorrect syntax near 'END'.
```

**Review by dnlnln — CHANGES_REQUESTED** [review]
Thanks for your contribution, I'm glad to hear you're getting good use out of it

Just left one comment for you, otherwise it looks good :)

**Comment by Dezleung** [comment]
for both binary and varbinary, it should use CONVERT(varchar(max), @column_name, 1) to generate HEX string as Source Value.

**Comment by nathfy** [comment]
Hi there @readyroll changes made, and @Dezleung I've updated the binary and varbinary conversions as you suggested. Take a look if you have time

Thanks

**Review by dnlnln — APPROVED** [review]

**Pull request merged**